src/main/java/org/springframework/samples/petclinic/owner/Owner.java

```diff
@@ -56,7 +56,7 @@ public class Owner extends Person {
 
 	@Column(name = "telephone")
 	@NotEmpty
```

Comment:
so the not empty is probably redundant?

Reply:
Don't know. What would be the effect of removing it? None of these errors has any error messages or nice handling associated with them anyway, so it's all kind of moot.